pandas/tests/indexes/datetimes/test_reindex.py

```diff
@@ -0,0 +1,25 @@
+from datetime import timedelta
+
+import pytest
+
+from pandas import NA, DataFrame, Index, date_range
+import pandas._testing as tm
+
+
+class TestReindex:
+    @pytest.mark.parametrize(
+        "kwargs",
+        [
+            {"method": "pad", "tolerance": timedelta(seconds=9)},
+            {"method": "backfill", "tolerance": timedelta(seconds=9)},
+            {"method": "nearest"},
+            {"method": None},
+        ],
+    )
+    def test_reindex_empty_frame(self, kwargs):
+        # GH#27315
+        idx = date_range(start="2020", freq="30s", periods=3)
+        df = DataFrame([], index=Index([], name="time"), columns=["a"])
+        result = df.reindex(idx, **kwargs)
```

Comment:
you're testing the DataFrame.reindex method, this goes in tests.frame.methods.test_reindex

rule of thumb: if you need Series or DataFrame, it probably doesnt belong in tests.indexes

Reply:
Thx, that helps me for the future. Moved the test